### PR TITLE
Run tests with test specific serviceaccounts and roles

### PR DESCRIPTION
**Pull request by lake-of-dreams**
# Description

Adds a new Jenkinsfile and related changes to be able to run certain tests with a serviceaccount having non cluster admin privs. Includes an unrelated change to expect `302` as a valid status on accessing kibana vmi.

Fixes VZ-2157

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: tests/e2e/pkg/kubernetes.go [hunk 1]
```diff
@@ -282,6 +291,21 @@ func GetNamespace(name string) (*corev1.Namespace, error) {
 
 // CreateNamespace creates a namespace
 func CreateNamespace(name string, labels map[string]string) (*corev1.Namespace, error) {
+
+	existingNamespace, err := GetNamespace(name)
+	if existingNamespace != nil && existingNamespace.Name == name {
+		return existingNamespace, nil
+	}
+
+	if errors.IsForbidden(err) && len(os.Getenv("TEST_KUBECONFIG")) > 0 {
```
Comment (gilbode) [inline tests/e2e/pkg/kubernetes.go:300]: This gives me concerns that we continue to execute commands but then ignore errors when running with less privileges.  I feel like this should be done differently, not entirely clear where/how these functions are called so don't have a concrete suggestion offhand.

Comment (lake-of-dreams) [reply tests/e2e/pkg/kubernetes.go]: fixed, basically check for `TEST_KUBECONFIG` as first thing and don't proceed with create ns/delete ns logic. For this to work, we need to give `get` permission to testrole

File: tests/e2e/pkg/kubernetes.go [hunk 2]
```diff
@@ -294,18 +317,26 @@ func CreateNamespace(name string, labels map[string]string) (*corev1.Namespace,
...
 	}
-	return ns, err
+
+	return ns, nil
 }
 
 // DeleteNamespace deletes a namespace
 func DeleteNamespace(name string) error {
+	if len(os.Getenv("TEST_KUBECONFIG")) > 0 {
+		Log(Info, fmt.Sprintf("DeleteNamespace %s, test is running with custom service account and therefore namespace won't be deletd by test", name))
```
Comment (markxnelson) [inline tests/e2e/pkg/kubernetes.go:329]: typo here - deleted

**Review by markxnelson — APPROVED** [review]

File: tests/e2e/pkg/kubernetes.go (hunk 2 above)
Comment (lake-of-dreams) [reply tests/e2e/pkg/kubernetes.go:329]: fixed in https://github.com/verrazzano/verrazzano/pull/758

**Pull request merged**